reference/supported-languages.md

```diff
@@ -0,0 +1,30 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
+* COBOL (Closed source, commercial feature)
+
+## Data formats
+
+* XML
+* Properties
+* YAML
+* JSON
+* Protobuf
+* Maven POMs (with dependency management)
+
+## Frameworks
+
+Framework developers have created recipes to help migrate and support their frameworks. These include:
```

Comment:
No idea if this is the right way of phrasing this. Would appreciate any advice on how this might be phrased differently.

Reply:
Is this sentence necessary? There isn't an equivalent one for Programming languages nor for Data formats.

Reply:
I was asked to call out that framework authors are responsible for these frameworks and made these rather than OpenRewrite/Moderne.

Reply:
I'd go with something like:
```suggestion
Framework migration recipes are developed and maintained in collaboration of the team behind OpenRewrite at Moderne, the original framework authors, and the wider OSS community. Individual recipes list the contributing authors. Support includes, but is not limited to:
```

This softly calls to action anyone looking to contribute, such that it's clear we're open to contributions. We also credit anyone that contributed to any recipe, such that there's no confusion who's involved.